Lib/idlelib/configdialog.py

```diff
@@ -1099,6 +1169,7 @@ def create_new(self, new_theme_name):
             save_new
             set_theme_type
         """
+
```

Comment:
Remove.  Please don't go around reformatting, especially is a way that counters existing consistent style.